server/server.go

```diff
@@ -433,6 +433,32 @@ func (s *Server) status(ctx context.Context, e lookout.Event, st lookout.Analysi
 	}
 }
 
+func (s *Server) dedupComments(comments []*lookout.Comment) []*lookout.Comment {
```

Comment:
I would better make it a method of `lookout.AnalyzerCommentsGroups` same as `Filter`.

Reply:
Good point! I agree 👍